.semaphore/semaphore.yml

```diff
@@ -32,10 +32,10 @@ blocks:
             - 'wget -O /tmp/libpng12.deb http://mirrors.kernel.org/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb'
             - sudo dpkg -i /tmp/libpng12.deb
             - gem install semaphore_test_boosters
-            - cache restore
+            # - cache restore
```

Comment:
Was this for testing? If so, we can undo this before we ship.

Reply:
yup semaphore + bootsnap was giving me issues...I re-enabled it.

Reply:
> yup semaphore + bootsnap was giving me issues...I re-enabled it.

semaphore cache is broken w/ this rails upgrade, not sure what is going on.  😢